data/human/news.txt

```diff
@@ -342,8 +370,6 @@ phrase "small trash item"
 		"cooling duct"
 		"radar jammer"
 		"fuel pod"
-		
-		
```

Comment:
These line removals deviate from the data format slightly I think? As these are related (the phrase and news item) I would suggest perhaps having 2 lines of space between the phrase and news items.